stories/object-status/object-status.stories.js

```diff
@@ -83,32 +83,32 @@ export const genericIndicationColors = () => `
  */
 
 export const clickableObjectStatus = () => `
-<a href="#" class="fd-object-status fd-object-status--link">Neutral</a>
-<a href="#" class="fd-object-status fd-object-status--negative fd-object-status--link">Negative</a>
-<a href="#" class="fd-object-status fd-object-status--positive fd-object-status--link">Positive</a>
-<a role="button" class="fd-object-status fd-object-status--critical fd-object-status--link"><i class="fd-object-status__icon sap-icon--status-critical"></i>Critical</a>
-<a role="button" class="fd-object-status fd-object-status--informative fd-object-status--link "><i class="fd-object-status__icon sap-icon--hint"></i>Informative</a>
+<a href="#" class="fd-object-status fd-object-status--link" aria-label="Neutral Status with clickable" aria-hidden="Neutral Status with clickable">Neutral</a>
```

Comment:
Same as before - node has text so we need to remove both `aria-hidden` and `aria-label`